astropy/modeling/functional_models.py

```diff
@@ -440,6 +440,99 @@ def inverse(self):
         return inv
 
 
+class Sersic1D(Fittable1DModel):
+    """
+    One dimensional Sersic surface brightness profile.
+
+    Parameters
+    ----------
+    amplitude : float
+        Central surface brightness, within r_eff. 
+    r_eff : float
+        Effective (half-light) radius
+    n : float
+        Sersic Index.
+
+    
+    See Also
+    --------
+    Gaussian1D, Moffat1D, Lorentz1D
+
+    Notes
+    -----
+    Model formula:
+
+    .. math::
+
+        I(r)=I_e\\exp\\left[-b_n\left(\\frac{r}{r_{e}}\\right)^{(1/n)}-1\\right]
+
+
+    The constant :math:`b_n` is defined such that :math:`r_e` contains half the total 
+    luminosity, and can be solved for numerically.  
+
+    .. math:: \\Gamma(2n) = 2\\gamma (b_n,2n)
```

Comment:
Have you checked, whether the  ´.. math::´  directive works when used inline? If not add a new line and indentation